pkg/backingstore/reconciler.go

```diff
@@ -1277,8 +1277,9 @@ func (r *Reconciler) updatePodTemplate() error {
 			[]corev1.LocalObjectReference{*r.NooBaa.Spec.ImagePullSecret}
 	}
 	r.PodAgentTemplate.Labels = map[string]string{
-		"app":  "noobaa",
-		"pool": r.BackingStore.Name,
+		"app":                 "noobaa",
+		"pool":                r.BackingStore.Name,
+		"noobaa-backingstore": r.Request.Name,
```

Comment:
why `noobaa-backingstore` added?

Reply:
We need to add a label to all the backingstore pods so that topology constraints should work. The current two labels would not work in here so I need to add one. Please correct me if I am wrong. Thanks